docs/solutions/connect_vcns_using_multiple_vnics/env-vars

```diff
@@ -0,0 +1,33 @@
+### Provider credentials
+export TF_VAR_tenancy_ocid=""
+export TF_VAR_user_ocid=""
+export TF_VAR_fingerprint=""
+export TF_VAR_private_key_path=""
+
+### Region
+export TF_VAR_region=""
+
+### AD
+export TF_VAR_AD=""
+
+### Compartment
+export TF_VAR_compartment_ocid=""
+
+### VCN-1 configuration
```

Comment:
I would remove all of these from vcn_cidr to InstanceOSVersion. The config sets good defaults for these, but if they are in your env-vars like this then they will get set to "" and the apply will fail. I suspect that most users will be fine with these defaults for an example like this.

Reply:
Wanted to give an option for users to change this in the env-vars if they wanted to.

Reply:
How about commenting these out and mark optional, like:
```
######
# Optional Variables
######

### AD
# Possible values are 1, 2, 3
# export TF_VAR_AD=""

### VCN-1 configuration
# export TF_VAR_vcn_cidr=""
# export TF_VAR_mgmt_subnet_cidr=""
# export TF_VAR_private_subnet_cidr=""

### VCN-2 configuration
# export TF_VAR_vcn_cidr2=""
# export TF_VAR_mgmt_subnet_cidr2=""
# export TF_VAR_private_subnet_cidr2=""

### Instance configuration
# export TF_VAR_InstanceShape=""
# export TF_VAR_InstanceOS=""
# export TF_VAR_InstanceOSVersion=""
```